### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by Lars-DHondt-KUL**
## Description

- Move settings overview to separate file.
- Organize documentation in a folder.

## Motivation and Context

This is cleaner.

## How Has This Been Tested?

n/a

## Suggested tests for reviewers

proofread

### CODE REVIEW

File: Documentation/SettingsOverview.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
+
+
```
Comment (VDB-Bram) [inline Documentation/SettingsOverview.md:2]:
```suggestion
Quickly navigate to:
* [Required Settings](SettingsOverview.md#required-settings)
* [Optional Settings](SettingsOverview.md#optional-settings)
  * [S.bounds](SettingsOverview.md#sbounds)
  * [S.metabolicE - metabolic-energy](SettingsOverview.md#smetabolice---metabolic-energy)
  * [S.misc - miscellanious](SettingsOverview.md#smisc---miscellanious)
  * [S.post_process](SettingsOverview.md#spost_process)
  * [S.solver](SettingsOverview.md#ssolver)
  * [Ssubject](SettingsOverview.md#ssubject)
  * [S.weights](SettingsOverview.md#sweights)
  * [S.OpenSimADOptions](SettingsOverview.md#sopensimadoptions)
  * [S.orthosis](SettingsOverview.md#sorthosis)

```

File: Documentation/SettingsOverview.md [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+    - always used for the quasi-random initial guess
+    - used for data-informed initial guess when `S.subject.adapt_IG_pelvis_y = 1;`
+    - S.subject.IG_pelvis_y is also used to establish bounds on vertical pelvis position.
+- **S.subject.adapt_IG_pelvis_y**: 
+    - boolean to adjust the trajectory of height of pelvis from the ground for data-informed initial guess. Default is *0*. 0 means the trajectory will not be changed. If 1, the trajectory will be changed such that the average value of the trajectory is equal to S.subject.IG_pelvis_y.
+- **S.subject.muscle_strength**: 
+    - structure with [scaling factors for muscle strength](./FiguresForDocumentation/fig_muscle_tendon_properties_scaling.png). This scales the max muscle force of the active muscle force. Default is *[]*, that is, no scaling. Input as a cell array where 1st input is the muscle(s) name, 2nd is the scale factor. If more than one scaling is to be performed, add 2 more inputs. For example, S.subject.muscle_strength = {{'soleus_l','soleus_r'},0.9,{'tib_ant_l'},1.1} will scale both soleus by a factor of 0.9 and tibialis anterior left by a scale of 1.1.
```
Comment (VDB-Bram) [inline Documentation/SettingsOverview.md:189]:
```suggestion
    - structure with [scaling factors for muscle strength](./FiguresForDocumentation/muscle_tendon_properties_scaling.png). This scales the max muscle force of the active muscle force. Default is *[]*, that is, no scaling. Input as a cell array where 1st input is the muscle(s) name, 2nd is the scale factor. If more than one scaling is to be performed, add 2 more inputs. For example, S.subject.muscle_strength = {{'soleus_l','soleus_r'},0.9,{'tib_ant_l'},1.1} will scale both soleus by a factor of 0.9 and tibialis anterior left by a scale of 1.1.
```

Comment (VDB-Bram) [inline Documentation/SettingsOverview.md:189]: figure is missing in the folder

Comment (Lars-DHondt-KUL) [reply Documentation/SettingsOverview.md:201]: oops! Updated the gitignore, now the figures should be there.

File: Documentation/SettingsOverview.md [hunk 3]
```diff
@@ -0,0 +1,310 @@
+
+Quickly navigate to:
+* [Required Settings](SettingsOverview.md#required-settings)
+* [Optional Settings](SettingsOverview.md#optional-settings)
+  * [S.bounds](SettingsOverview.md#sbounds)
+  * [S.metabolicE - metabolic-energy](SettingsOverview.md#smetabolice---metabolic-energy)
+  * [S.misc - miscellanious](SettingsOverview.md#smisc---miscellanious)
+  * [S.post_process](SettingsOverview.md#spost_process)
+  * [S.solver](SettingsOverview.md#ssolver)
+  * [Ssubject](SettingsOverview.md#ssubject)
```
Comment (VDB-Bram) [inline Documentation/SettingsOverview.md:10]:
```suggestion
  * [S.subject](SettingsOverview.md#ssubject)
```

**Review by VDB-Bram — APPROVED** [review]
Proofread.
All comments are addressed.

**Pull request merged**